src/components/MaterialUI/muiThemes.js

```diff
@@ -7,9 +7,15 @@ const commonTheme = {
 	spacing: factor => `${0.625 * factor}rem`,
 	direction: "ltr",
 	shape: {
-		borderRadius: 4,
+		borderRadius: 4
 	},
 	typography: {
+		h1Size: 24,
+		h2Size: 18,
+		h3Size: 16,
+		h4Size: 14,
+		h5Size: 13,
+		h6Size: 13,
```

Comment:
is it here right for 13? And I don't see 12px that is already used in ExpansionPanel

Reply:
@Anstsiya I noticed it was missing when I sent the PR, will add it today. It will be called fieldLabelSize so you could start using it.